src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,42 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected LocalDate by;
```

Comment:
`private` should be a better accesser here than `protected` because there are no other classes extending from Deadline, therefore, the only object that will access this field is of type `Deadline`. Declaring this field as `final` should also be a good practice because the time should not be changed after `new()` an object.